doc/examples/grant-metadata-schema.json

```diff
@@ -0,0 +1,74 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
where do we plan to host the schemas?  how will we version them, so if we want to update them we can?

Reply:
Good questions @owocki! Correct me if I'm wrong @phutchins but I think this was checked in as a starting point for the conversation, since Chainsafe put it together. I think we plan to work through these kinds of details starting this week as get the metadata storage part of the project moving.

Reply:
Yes, this is intended to be a place to start. I've added readmes clarifying the purpose and added structure intended to house the protocol definition.